pkg/internal/cmd/cmd.go

```diff
@@ -70,25 +98,31 @@ func NewCommand(ctx context.Context) *cobra.Command {
 					BindAddress: opts.MetricsAddress,
 				},
 				WebhookServer: ctrlwebhook.NewServer(ctrlwebhook.Options{
-					Port:    opts.Webhook.Port,
-					Host:    opts.Webhook.Host,
```

Comment:
It seems like this option is no longer used, or? Maybe clean up the Helm chart if that's the case?

Reply:
Sorry, I missed this. Also, I missed some other parts, the PR should be ready now.